### DISCUSSION TITLE
Relax botocore version dependency

### DISCUSSION THREAD

**Discussion by thenorthnate** [aio-libs/aiobotocore#960] [Q&A] [answered]
Hi Team,

I wanted to echo #920 specifically with regard to how the version is pinned. Is it possible to relax the constraints for botocore and boto3 to be something like:

`botocore~=1.24.21` and `boto3~=1.21.21`

This way security fixes to the boto libraries can be used without having to have the exact patch release as well? We are using this library (thank you very much for maintaining this!) in a fairly complex repository that also uses botocore and boto3 and has other restrictions on those libraries. Relaxing the patch release version of boto3 and botocore would fix the incompatibility issues we have.

Thank you!

**Comment by thehesiod** [comment, marked as answer]
in this example: https://github.com/aio-libs/aiobotocore/pull/974 it's a patch however will involve substantial changes to incorporate the new async paths, so answer is no
